unime/src-tauri/tests/fixtures/states/credential_redirect_me.json

```diff
@@ -8,7 +8,7 @@
     "credentials": [
         {
             "id": "39373933-3863-3339-3864-646234373631",
-            "issuer_name": null,
+            "issuer_name": "",
```

Comment:
Is an empty string really better than `null`? Null can be more easily checked against and an empty string would be valid, but not display anything. Is this desired behavior?

Reply:
Yes I also discussed with @nanderstabel  that issuer_name is always set in the code, so it can never be null anyway. We need to make a new PR in the future where we look at the frontend ?? checks on the issuer_name to incorporate those back to the backend again.